modules/configuration/src/main/java/org/apache/ignite/internal/configuration/ConfigurationChanger.java

```diff
@@ -302,19 +274,18 @@ public void initialize(ConfigurationType storageType) throws ConfigurationValida
                 )
             );
         }
-
-        ConfigurationStorage actualStorage = storageInstances.get(storagesTypes.iterator().next());
-
-        if (storage != null && storage != actualStorage) {
+        else if (storagesTypes.iterator().next() != storage.type()) {
             return CompletableFuture.failedFuture(
                 new ConfigurationChangeException("Mismatched storage passed.")
             );
         }
 
-        return changeInternally(source, actualStorage);
+        return changeInternally(src);
     }
 
-    /** Stop component. */
+    /**
+     * Stop component.
```

Comment:
I saw that you changes multi-line comments to single-line somewhere (in RootKey, I think). What was your motivation?